src/views/TechnicalAssistance/AdviceLetterForm/Recommendations.tsx

```diff
@@ -1,75 +1,167 @@
 
   /** Whether recommendations have been added to the request */
   const hasRecommendations: boolean = recommendations.length > 0;
 
-  return (
-    <div id="trbAdviceRecommendations">
-      {/* Add recommendation button */}
-      <Button
-        className="margin-top-5 margin-bottom-1"
-        type="button"
-        onClick={() => null}
-      >
-        {t('adviceLetterForm.addRecommendation')}
-      </Button>
+  const formMethods = useForm<AdviceLetterRecommendationFields>({
+    resolver: yupResolver(adviceRecommendationSchema),
+    defaultValues
+  });
+  const { reset } = formMethods;
 
+  const [remove, { loading }] = useMutation<
+    DeleteTRBRecommendation,
+    DeleteTRBRecommendationVariables
+  >(DeleteTrbRecommendationQuery, {
+    refetchQueries: [
       {
-        /* No recommendations message */
-        !hasRecommendations ? (
-          <Alert type="info" slim>
-            {t('adviceLetterForm.noRecommendations')}
-          </Alert>
-        ) : (
-          /* Recommendations list */
-          <ul>
-            {recommendations.map(({ title, id }) => (
-              <li key={id}>{title}</li>
-            ))}
-          </ul>
-        )
+        query: GetTrbAdviceLetterQuery,
+        variables: {
+          id: trbRequestId
+        }
       }
+    ]
+  });
+
+  useEffect(() => {
+    setIsStepSubmitting(loading);
+  }, [setIsStepSubmitting, loading]);
 
-      {/* Form pager buttons */}
-      <Pager
-        className="margin-top-4"
-        back={{
-          outline: true,
-          onClick: () => {
-            history.push(`/trb/${trbRequestId}/advice/summary`);
+  return (
+    <Switch>
+      {/* Recommendations Form */}
+      <Route exact path={`${path}/form`}>
+        <FormProvider {...formMethods}>
+          <RecommendationsForm
+            trbRequestId={trbRequestId}
+            setFormAlert={setFormAlert}
+          />
+        </FormProvider>
+      </Route>
+
+      {/* Recommendations form step */}
+      <Route exact path={`${path}`}>
+        <div id="trbAdviceRecommendations">
+          {/* Add recommendation button */}
+          <Button
+            className="margin-top-5 margin-bottom-1"
+            type="button"
+            onClick={() => {
+              reset(defaultValues);
+              history.push(`${url}/form`);
+            }}
+            outline={hasRecommendations}
+          >
+            {t(
+              hasRecommendations
+                ? 'adviceLetterForm.addAnotherRecommendation'
+                : 'adviceLetterForm.addRecommendation'
+            )}
+          </Button>
+
+          {
+            /* No recommendations message */
+            !hasRecommendations ? (
+              <Alert type="info" slim>
+                {t('adviceLetterForm.noRecommendations')}
+              </Alert>
+            ) : (
+              <RecommendationsList
+                type="form"
+                recommendations={recommendations}
```

Comment:
This could probably use some grid responsiveness.  On mobile it can get a bit scrunched